openfga/v1/errors_ignore.proto

```diff
@@ -4,8 +4,19 @@ package openfga.v1;
 
 import "protoc-gen-openapiv2/options/annotations.proto";
 
-enum ErrorCode {
+enum AuthErrorCode {
+  no_auth_error = 0;
+
+  auth_failed_invalid_subject = 1001;
+  auth_failed_invalid_audience = 1002;
+  auth_failed_invalid_issuer = 1003;
+  invalid_claims = 1004;
+  auth_failed_invalid_bearer_token = 1005;
+  bearer_token_missing = 1010;
+  unauthorized = 1012;
```

Comment:
Unauthorized should be under a separate category, since these are reserved for unauthenticated error codes.

Reply:
Whoops, that is a typo that came over from OpenFGA. It should be unauthenticated. Fixed. Thank you!